src/relay/backend/aot_executor_codegen.cc

```diff
@@ -374,13 +374,14 @@ class AOTExecutorCodegen : public ExprVisitor {
     }
 
     auto pf0 = GetPackedFunc("relay.backend._make_CCacheKey");
-    auto pf1 = GetPackedFunc("relay.backend._CompileEngineLower");
+    auto pf1 = GetPackedFunc("relay.backend._CompileEngineLowerWithModuleName");
```

Comment:
Why do we need a separate function ? Isnt it possible to improve _CompileEngineLower to accept module name generally ?